clippy_lints/src/non_copy_const.rs

```diff
@@ -49,7 +62,7 @@ declare_clippy_lint! {
     /// assert_eq!(STATIC_ATOM.load(SeqCst), 9); // use a `static` item to refer to the same instance
     /// ```
     pub DECLARE_INTERIOR_MUTABLE_CONST,
-    correctness,
+    style,
```

Comment:
```suggestion
    style, // FIXME: this is a correctness problem but there's no suitable warn-by-default category.
```
As flip1995 suggested, I think it's better to put two short comments next to the wrong categorization.

Reply:
I tried adding the comments there before but running cargo dev update_lints would end up removing the lint from the lint list. Is there a way to place comments there so that it doesn't interfere with the declare_clippy_lint macro?

Reply:
You can just put it in front of the macro call.